Answers.md

```diff
@@ -1,5 +1,14 @@
 1.  Explain the differences between `client-side routing` and `server-side routing`.
+    Client side routing takes place within the web browser and typically does not need to consult a server at all or multiple times. Server side routing needs a constant connection to a server and makes frequent requests
+
 1.  What does HTTP stand for?
+    hypertext transfer protocol
+
 1.  What does CRUD stand for?
+    Create, Read, Update, Delete
+
 1.  Which HTTP methods can be mapped to the CRUD acronym that we use when interfacing with APIs/Servers.
-1.  Mention three tools we can use to make AJAX requests
\ No newline at end of file
+    get, post, put, delete
```

Comment:
In the future, make sure to pair your response with the acronym, ie Create => Post, etc.